src/pages/ValidateConfigFiles/ServiceTemplatesTab.js

```diff
@@ -30,23 +30,32 @@ class EditTemplateFile extends Component {
 
   componentWillMount() {
     this.setState({ loading: true });
+    this.loadFileContent();
+  }
+
+  componentWillReceiveProps(props) {
+    if(this.props.fileLoadTime !== props.fileLoadTime){
+      loadFileContent();
+    }
+  }
+
+  loadFileContent = () => {
     fetchJson('/api/v2/service/files/' +  this.props.editFile)
-      .then((response) => {
```

Comment:
Not really sure what happened here. It looks like the tabbing got a bit messed up.

Reply:
ran lint on it and updated spacing to match recommendations from there